ztp/ztp-ran-policy-generator/ranGenTemplateExamples/site-du-sno-1-ranGen.yaml

```diff
@@ -49,17 +49,10 @@ sourceFiles:
     name: "perfprofile-policy"
     spec:
       # ex: overriding the additionalKernelArgs values exist in the source policy file PerformanceProfile
-      additionalKernelArgs:
-      - "nosmt"
-      - "nmi_watchdog=0"
-      - "audit=0"
-      - "mce=off"
-      - "processor.max_cstate=1"
-      - "idle=poll"
-      - "intel_idle.max_cstate=0"
+      additionalKernelArgs: []
```

Comment:
you don't need to add additionalKernelArgs as it is already defined in source PerformanceProfile.yaml  as additionalKernelArgs: []